src/components/ResultContainer.tsx

```diff
@@ -268,11 +268,16 @@ function ResultContainer({
     try {
       const subjectsResponse = await onDownload(buttonIndex, download);
       setDownloadError(null);
+      const timestamp = new Date()
+        .toISOString()
+        .replace('T', '-')
+        .replace(/:/g, '-')
+        .replace(/\..+/, '');
```

Comment:
I suggest changing the format of the timestamp to the following: YYYYMMDDHHMMSS for a less-clutered and cleaner file name. Wdyt?